src/simulation.c

```diff
@@ -170,15 +172,24 @@ void simulation_history_to_file(Simulation *sp) {
   } else {
     mkdir(path,0777);
   }
-  sprintf(end, "/%ld", sp->seed);
-  FILE *file = fopen(path,"w");
+
+  char time_path[2048];
+  char* time_end = strcpy(time_path, path);
+  time_end += strlen(time_path);
+
+  sprintf(end, "/reactions_%ld", sp->seed);
+  sprintf(time_end, "/times_%ld", sp->seed);
+  FILE *reaction_file = fopen(path,"w");
+  FILE *time_file = fopen(time_path,"w");
   int i;
   Chunk *chunk = sp->history->first_chunk;
   while (chunk) {
-    for (i = 0; i < chunk->next_free_index; i++)
-      fprintf(file, "%d\n", chunk->data[i]);
+    for (i = 0; i < chunk->next_free_index; i++) {
+      fprintf(reaction_file, "%d\n", chunk->data[i].reaction);
+      fprintf(time_file, "%.17g\n", chunk->data[i].time);
```

Comment:
I've been using "%e\n" for printing floats

Reply:
Since these numbers are so damn small, I want to make sure we're getting full precision, hence the %.17. I think scientific notation could cause problems in some places (for instance, I had problems when the number of steps was given in scientific notation), so probably %.17f is actually better.